src/core/tests/preprocess.cpp

```diff
@@ -209,6 +212,129 @@ TEST(pre_post_process, tensor_element_type_and_scale) {
     EXPECT_EQ(ov::layout::get_layout(f->input(0)), Layout());
 }
 
+class pre_post_process_to_gray : public ::testing::TestWithParam<ColorFormat> {};
```

Comment:
Detail: Consider rename this class to use `CamelCase` will be more consistent with other class names. e.g `ToGrayTest`
Then final test name in log will look like:
`pre_post_process.ToGrayTest. ....` instead `pre_post_process.pre_post_process_to_gray. ....` there will beno `pre_post_process` duplication.

Reply:
Renamed, but I used snake_case for compatibility with other names